Install, upgrade and remove specific package from device_list.fact file

fheslouin opened the pull request:
These changes introduce the possibility to install, upgrade and remove package.

We now have a list of package where each one has the fellowing status : 
- `installed` : AnsibleCube will ensure the package is installed on the device
- `updated` : AnsibleCube will ensure the package will be updated each time Internet is available if already installed, or installed and next time update if not present on the device yet
- `removed` : AnsibleCube will ensure the package is not present on the device any more

I have also change `zim_install` role name to `package_management` which is more generic as we now install zim file, package and static-site through catalog.

Any comments are welcome

Commit:
Limit the maximum number of stations allowed in station table

Commit:
Checks are now done in the role

Commit:
Add a status variable for each package

Commit:
Install, upgrade or remove a package

Fellowing the value given in each status package, install, upgrade or
remove the latter.

Jinja filter has been used to select the right command to be done.
Since the command "ideascube catalog install package" just install the
package, for a package update, in all case,  we have first to install the package then
update it. It would be easier if ideascube catalog install could install
and upgrade the package if the latter is already installed.

Commit:
Change zim_install role name to package_management

Commit:
Rename folder

letompouce commented:
I might be a little pushy here, but this `installed`/`removed`/`updated` taxonomy could be aligned with ansible's `present`/`absent`/`latest`.

letompouce reviewed (commented):
How comes the new `status` attribute is a list?

fheslouin commented:
> could be aligned with ansible's present/absent/latest.

Good idea, I'll modify that.

> How comes the new status attribute is a list?

Humm, good point, there is no reason actually, except if we would like to give more variables like, present, latest, etc. But I'm not certain of the needs

Commit:
status does not need to be a list, change taxonomy

installed/removed/updated has been swap for present/absent/latest

Commit:
change taxonomy for status

Commit:
Speed up check of installed/updated/removed pkg

Because the ideascube CLI takes some time to be executed each time
ansiblecube check if a package is installed to be updated or removed,
we directly verify the content of /var/ideascube/main/catalog/installed.yml
This file contain all the installed package and thus can be check
directly instead of using the ideascube CLI.

Commit:
Add support of package_management in buildMyCube

* Building of JSON schema has been adapted to support nested arrays in objects.
* If previous configuration file existe (device-list) ask rather to
  delete it or keep it

Commit:
Kiwix server v1.0 does not exist yet for x86

Commit:
Erase previous configuration

Commit:
Clear ideascube cache after install succeded

Commit:
Add language en for Kalite on kb-fra-crfaix

Commit:
Install Kiwix v1.0 on kb-cod-kinlab

Commit:
set w2eu package to latest state

Commit:
Add package help.unhcr.org to idb-grc-ioannina

Commit:
Add package help.unhcr.org to kb-grc-ioannina

Commit:
Add package help.unhcr.org to kb-grc-konitsa
